static/styles/homepage/header.scss

```diff
@@ -0,0 +1,40 @@
+.header{
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: var(--color-white);
+    color: var(--color-black);
```

Comment:
Wir brauchen für die CSS Variablen Fallbacks, weil Landingpage und wir nicht spezielle Browser enforcen können. https://stackoverflow.com/questions/44271920/css-variables-with-fallback-for-older-browsers

Reply:
https://codepen.io/jakealbaugh/post/css4-variables-with-fallbacks-using-sass

Reply:
Einfachste Version:
color: #fff 
color: var(--color-black)

Reply:
> Einfachste Version:
> color: #fff
> color: var(--color-black)

Geht glaube per präprozessor wie postcss oder so zu lösen